tests/cypress/support/accessibility.js

```diff
@@ -1,35 +1,50 @@
 import "cypress-file-upload";
 import "@cypress-audit/pa11y/commands";
 
+const breakpoints = {
+  mobile: [560, 800],
+  tablet: [880, 1000],
+  desktop: [1200, 1200],
+};
+
+export const checkCurrentRouteAccessibility = () => {
+  Object.keys(breakpoints).forEach((deviceSize) => {
+    const size = breakpoints[deviceSize];
+    it(
+      `Has no basic accessibility issues on ${deviceSize}`,
+      { retries: 0 },
+      () => {
+        cy.viewport(...size);
+        cy.runAccessibilityTests();
+      }
+    );
+  });
+};
```

Comment:
This method is called in each of the route-specific `.spec` files. It creates and triggers a full accessibility test for each of the 3 main breakpoints.